opentelemetry/datadog.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Integrating pganalyze with Datadog APM'
+backlink_href: /docs/opentelemetry
+backlink_title: 'OpenTelemetry Integration'
+---
+
+You can set up the pganalyze collector OpenTelemetry exporter with Datadog APM to
+show spans for slow query EXPLAIN plans inside your traces.
+The span contains the link to the EXPLAIN plan in the pganalyze app, so that you
+can easily visit the pganalyze page to find out why the query is slow.
+
+![Screenshot of Datadog APM integration](./datadog.png)
+
+Compared to Datadog DBM, pganalyze gives you exact details on the specific query execution,
+by combining `auto_explain` data extracted from the Postgres logs, with a specific trace.
```

Comment:
```suggestion
Compared to Datadog DBM, pganalyze gives you exact details on the specific query execution
by combining `auto_explain` data extracted from the Postgres logs with a specific trace.
```
I'm shaky on proper comma usage, but my gut feeling is these should be omitted. Take it or leave it.